modules/ROOT/partials/configuration/content_langs.adoc

```diff
@@ -5,7 +5,7 @@ include::partial$misc/requires_5_9v.adoc[]
 
 include::partial$configuration/content_langs_base.adoc[]
```

Comment:
The table in this file needs some formatting fixes, as most of the content is in the last column. I'd suggest adding `[cols="1,1,1,4"]` above the table (this is what we did in the 6.x docs).

Reply:
Done